mne/decoding/csp.py

```diff
@@ -346,9 +347,9 @@ def plot_patterns(self, info, components=None, ch_type=None, layout=None,
             a circle will be used (via .add_artist). Defaults to True.
         colorbar : bool
             Plot a colorbar.
-        scale : dict | float | None
-            Scale the data for plotting. If None, defaults to 1e6 for eeg, 1e13
-            for grad and 1e15 for mag.
+        scalings : dict | float | None
```

Comment:
can scalings also be 'auto' here, or is that just for the raw plot?

Reply:
I don't think so, I think it's unique to `raw` currently